src/System.Threading.Thread/src/System/Threading/Thread.Unix.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace System.Threading
 {
-    public sealed partial class Thread : CriticalFinalizerObject
+    public sealed partial class Thread
```

Comment:
It still derives from CriticalFinalizerObject in Thread.cs.  There is no need to specify the base class in all three Thread*.cs files; one specification suffices.